charts/argocd-apps/values-test.yaml

```diff
@@ -36,3 +36,47 @@ applications:
       value: "1"
     - name: worker.replicas
       value: "1"
+- name: router
+  helm:
+    parameters:
+    - name: govukEnvironment
+      value: "test"
```

Comment:
Also, how do we feel about encapsulating Helm values files inside a Helm values file like this? Seeing it in practice, it seems kinda unnecessarily meta to me. I still think it the idea of making the values part of the Application resource sounds attractive in itself, but I hadn't anticipated this rather ugly consequence of it.